test/sslapitest.c

```diff
@@ -10043,6 +10043,26 @@ static int test_load_dhfile(void)
 #endif
 }
 
+static int test_calling_set1_groups_list_twice(void)
+{
+    int testresult = 0;
+
+    SSL_CTX *ctx = NULL;
+
+    if (!TEST_ptr(ctx = SSL_CTX_new_ex(libctx, NULL, TLS_method())))
+        goto end;
+
+    /* calling SSL_CTX_set1_groups_list twice to ensure, it's not leaking memory */
+    SSL_CTX_set1_groups_list(ctx, "secp384r1:prime256v1");
```

Comment:
I'd wrap these two calls in `if(!TEST_true(SSL_CTX_set1_groups_list(...)))`

Reply:
done